tar.go

```diff
@@ -117,10 +115,7 @@ func (ts *tarStream) readHeaders() {
 				if hdr.FileInfo().IsDir() && keyword == "size" {
 					continue
 				}
-
-				if string(hdr.Typeflag) == string('1') {
-					// TODO: get number of hardlinks for a file
-				}
+				// TODO: handle hardlinks
```

Comment:
yep, #40 talks about the handling of some more keywords, of which includes `nlink`